src/components/Calculator.css

```diff
@@ -0,0 +1,44 @@
+.mayorContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 1rem;
+  height: 100vh;
+}
+
+.calContainer {
+  width: 50%;
+  height: 50%;
+}
+
+.result-container {
+  display: flex;
+  justify-content: flex-end;
+  align-items: flex-end;
+  background-color: rgb(165, 163, 163);
+}
+
+.result {
+  color: #fff;
+  padding: 1rem;
+}
+
+.keyContainer {
+  display: grid;
+  justify-items: stretch;
+  align-items: stretch;
+  align-content: stretch;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: auto auto auto auto auto;
+  height: 100%;
+}
+
+.my-0 {
+  grid-column-start: 1;
+  grid-column-end: 3;
+}
+
+.operator {
+  background-color: rgb(243, 133, 105);
+}
```

Comment:
- [ ] Your calculator is almost identical to that of the design layout but the color of the _**mathematical symbols**_ does not match that of the template given. Kindly change the color of the class `.operator` so that it matches that of the design. You can use this color `#f5913e` or a color closer to it.

### Design Template

![image](https://user-images.githubusercontent.com/77106962/158674136-48ec0d19-48cf-4235-a64c-aa99a31e7071.png)

### Your Design

![image](https://user-images.githubusercontent.com/77106962/158674203-ad01570f-de44-4a23-ad10-678c0e83bbb0.png)